src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyServiceTest.kt

```diff
@@ -233,6 +234,7 @@ class NotifyServiceTest {
       variationForReApprovalTemplateId = TEMPLATE_ID,
       unapprovedLicenceByCrdTemplateId = TEMPLATE_ID,
       client = notificationClient,
+      internalEmailAddress = "testemail@probation.gov.uk"
```

Comment:
probably want to extend the `send unapproved licence email to probation practitioner` to ensure you are sending to the new email address

Reply:
Ah did you mean replace it with the `EMAIL_ADDRESS` constant that I had missed previously? 😅

Reply:
That test is presumably only verifying that its sending emails to the PP. Should also now add checks that we're sending email to the internal email address?

Reply:
Oh I see! Gotcha, yes I will do.
Also I'm having one of 'those' mornings, of course I don't want to use the existing `EMAIL_ADDRESS`, the whole point is that it's a different one 🤦